### ISSUE TITLE
Use MenuButton instead of custom controls

### ISSUE THREAD

**Issue by Awkor** [LittleMouseGames/whiskers#13]
Currently the menu bar buttons use custom controls to achieve a similar behavior to [MenuButton](https://docs.godotengine.org/en/3.0/classes/class_menubutton.html).

Here's how the MenuButton looks like in the Godot editor:
![20190210](https://user-images.githubusercontent.com/19125153/52535378-f81ea380-2d4d-11e9-923a-6df02006a866.png)

We should consider using the MenuButton node because it would:
* Simplify the tree scene (the addition of menu items is done through code)
* Ease the handling of shortcuts (MenuButton uses a [PopupMenu](https://docs.godotengine.org/en/3.0/classes/class_popupmenu.html#class-popupmenu) which allows to specify a [ShortCut](https://docs.godotengine.org/en/3.0/classes/class_popupmenu.html#class-popupmenu-set-item-shortcut) to each item)
* Remove the need to manage the opening and closing of each menu (it's handled by default)
* Allow to use the `switch_on_hover` property

The only caveat is that there's currently an [issue](https://github.com/godotengine/godot/issues/25519) that hides a menu item label when a shortcut is set.

Thoughts?

**Comment by yochrisbolton** [comment]
I agree that we should switch to MenuButton, however that bug you linked looks like it could cause some headaches, and isn't slated to be fixed until 3.2 which is a little concerning.

I'm curious if there is an easy workaround so that we can switch to the MenuButton button nodes and still add shortcuts. I also wonder if the bug is only in effect when programmatically adding items / labels?
